docs/key-constraints.rst

```diff
@@ -4,23 +4,27 @@
 Message Keys
 ============
 
-The `CREATE STREAM` and `CREATE TABLE` statements allow the user to specify the column that corresponds to the Kafka message key by setting the `KEY` property of the `WITH` clause. For example:
+The ``CREATE STREAM`` and ``CREATE TABLE`` statements, which read data from a Kafka topic into a stream or table, allow you to specify a field/column in the Kafka message value that corresponds to the Kafka message key by setting the ``KEY`` property of the ``WITH`` clause.
```

Comment:
This still doesn't sound right to me.  I thought the ``KEY`` property allowed a user to map the key of the message in Kafka to a column in the KSQL stream's/table's schema.

Reply:
> I thought the KEY property allowed a user to map the key of the message in Kafka to a column in the KSQL stream's/table's schema.

Alas no.  It would be great if it did, see https://github.com/confluentinc/ksql/issues/804.  The key must currently be present in BOTH the message key and "somewhere" in the message value as well (= "duplicate key" requirement).